src/app/page.tsx

```diff
@@ -243,89 +266,89 @@ export default function Home() {
             Gold
           </h3>
           <hr className="w-11/12 border-t-2 border-gray-500 mb-10 ml-5 mr-5" />
-          <div className="flex flex-col md:flex-row items-center mb-8">
+          <div className="flex flex-col justify-evenly md:mx-2 md:flex-row items-center mb-8">
             <Image
               width={163}
               height={76}
               src="icaf-logo-transparent.svg"
               alt="ICAF logo"
-              className="md:ml-5 w-5/6 md:w-1/4 mb-4 md:mb-0"
+              className="w-5/6 md:w-1/4 mb-4 md:mb-0"
             />
             <Image
               width={163}
               height={76}
               src="icaf-logo-transparent.svg"
               alt="ICAF logo"
-              className="md:ml-10 w-5/6 md:w-1/4 mb-4 md:mb-0"
+              className="w-5/6 md:w-1/4 mb-4 md:mb-0"
             />
             <Image
               width={163}
               height={76}
               src="icaf-logo-transparent.svg"
               alt="ICAF logo"
-              className="md:ml-10 md:mr-10 md:w-1/4 w-5/6 md:mb-0"
+              className="md:w-1/4 w-5/6 md:mb-0"
             />
           </div>
         </div>
         <div className="col-span-4 flex flex-col items-center bg-neutral-white rounded-2xl mr-5 mb-10">
-          <h3 className="break-words font-semibold text-3xl text-main-silver mt-10 mb-5">
+          <h3 className="break-words font-semibold text-xl md:text-3xl text-main-silver mt-10 mb-5">
             Silver
           </h3>
           <hr className="w-11/12 border-t-2 border-gray-500 mb-10 ml-5 mr-5" />
-          <div className="flex flex-col md:flex-row items-center">
+          <div className="flex flex-col justify-evenly md:mx-2 md:mt-2 md:flex-row items-center">
             <Image
               width={163}
               height={76}
               src="icaf-logo-transparent.svg"
               alt="ICAF logo"
-              className="md:ml-10 md:w-1/5 w-4/5 mb-4 md:mb-0"
+              className="md:w-1/5 w-4/5 mb-6 md:mb-0"
             />
             <Image
               width={163}
               height={76}
               src="icaf-logo-transparent.svg"
               alt="ICAF logo"
-              className="md:ml-10 md:w-1/5 w-4/5 mb-4 md:mb-0"
+              className="md:w-1/5 w-4/5 mb-6 md:mb-0"
             />
             <Image
               width={163}
               height={76}
               src="icaf-logo-transparent.svg"
               alt="ICAF logo"
-              className="md:ml-10 md:w-1/5 w-4/5"
+              className="md:w-1/5 w-4/5"
             />
           </div>
         </div>
         <div className="col-span-3 flex flex-col items-center bg-neutral-white rounded-2xl mr-5 md:mr-20 mb-10">
-          <h3 className="break-words font-semibold text-3xl text-main-bronze mt-10 mb-5">
+          <h3 className="break-words font-semibold text-xl md:text-3xl text-main-bronze mt-10 mb-5">
```

Comment:
I still see the logos falling outside of the content box (for example at a breakpoint of `800px`). You might want to use `flex-wrap` to take care of this. This might end up looking different than the design, but the design is clearly not responsive anyway.